src/Makefile.gtest.include

```diff
@@ -9,6 +9,7 @@ zcash_gtest_SOURCES = \
 	gtest/test_checktransaction.cpp \
 	gtest/json_test_vectors.cpp \
         gtest/json_test_vectors.h \
+	gtest/test_fillnewblock.cpp \
```

Comment:
Yes, test_checktransaction has to run before the tests that set Params. Since you're updating the file worth putting a comment next to test_checktransaction to remind us not to move it from the top.